test/types/model/Query.ts

```diff
@@ -13,6 +14,14 @@ async function queryExecUnTyped (): Promise<AnyDocument[]> {
 	return await UserModel.query().exec();
 }
 
+async function queryExecUnTypedQueryResponse (): Promise<QueryResponse<AnyDocument>> {
+	return await UserModel.query().exec();
+}
+
+async function queryExecTyped (): Promise<QueryResponse<User>> {
+	return UserTypedModel.query("name").eq("Will").exec();
+}
```

Comment:
Let's keep it consistent. I know the ones above use functions, but that should probably be changed.

```suggestion
const queryExecUnTypedQueryResponse: Promise<QueryResponse<AnyDocument>> = UserModel.query().exec();
const queryExecTyped: Promise<QueryResponse<User>> = UserTypedModel.query("name").eq("Will").exec();
```